roslibrust/examples/basic_publisher.rs

```diff
@@ -1,19 +1,16 @@
-use log::*;
-use roslibrust::ClientHandle;
-
+#[cfg(feature = "rosbridge")]
 roslibrust_codegen_macro::find_and_generate_ros_messages!("assets/ros1_common_interfaces/std_msgs");
 
 /// This example creates a client, and publishes a message to the topic "talker"
 /// Running this example at the same time as subscribe_and_log will have the two examples
 /// pass messages between each other.
 /// To run this example a rosbridge websocket server should be running at the deafult port (9090).
+#[cfg(feature = "rosbridge")]
 #[tokio::main(flavor = "multi_thread")]
-async fn main() -> Result<(), anyhow::Error> {
-    simple_logger::SimpleLogger::new()
-        .with_level(log::LevelFilter::Debug)
-        .without_timestamps() // required for running wsl2
-        .init()
-        .unwrap();
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    use log::*;
+    use roslibrust::rosbridge::ClientHandle;
+    env_logger::init();
```

Comment:
Similar question here, though it looks like you replaced simple_logger with env_logger everywhere- is env_logger better?

Reply:
I've found env_logger to the most common "starting out logger."

I can't remember exactly why I started out with simple_logger, but subsequently have found env_logger to just be a lot more commonly used, so this seemed like a slightly "happier path" for the examples.

Reply:
I just did a check since I was curious:

7.5M downloads per month: https://lib.rs/crates/env_logger
1M downloads per month: https://lib.rs/crates/simple_logger

So it isn't like simple_logger isn't used, but env_logger is ~7x more used.

Reply:
env_logger is more flexible to configure but has the common pitfall of not piping to stdout/stderr unless you specify RUST_LOG env variable and some people find that as a hurdle for simple logging.